packages/color-field/stories/color-field.stories.ts

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import { html, TemplateResult } from '@spectrum-web-components/base';
+
+import '../sp-color-field.js';
+
+export default {
+    component: 'sp-color-field',
+    title: 'Color Field',
+};
+
+export const Default = (): TemplateResult => {
+    return html`
```

Comment:
why a Default color-field have a devMode prop? Do we in contract with the API to push this to production?

Reply:
I think you are right and in 'default' this devMode should not be present